### ISSUE TITLE
[change] Optimize obtain & validate token views

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-radius#257]
While working on https://github.com/openwisp/openwisp-radius/pull/256 I realized these two views are generating far too many queries, we should optimize these 2 views.

**Comment by ujjwalkirti** [comment]
Hello @nemesifier I want to work on this issue, can I proceed?

**Comment by nemesifier** [comment]
> Hello [@nemesifier](https://github.com/nemesifier) I want to work on this issue, can I proceed?

I don't think this task is suited to new contributors. You're welcome to contribute on easier tasks.

**Comment by ujjwalkirti** [comment]
Okay, thanks for responding @nemesifier .
